src/main/java/com/glitchcog/fontificator/config/ConfigEmoji.java

```diff
@@ -740,6 +759,7 @@ public void deepCopy(ConfigEmoji copy)
         this.ffzEnabled = copy.ffzEnabled;
         this.twitchLoaded = copy.twitchLoaded;
         this.twitchBadgesLoadedChannel = copy.twitchBadgesLoadedChannel;
+        this.twitchBadgesLoadedChannel = copy.twitchBadgesLoadedGlobal;
```

Comment:
Cut and paste bug; you mean: `this.twitchBadgesLoadedGlobal = copy.twitchBadgesLoadedGlobal;`